zerver/webhooks/grafana/view.py

```diff
@@ -33,8 +35,18 @@ def api_grafana_webhook(
     if payload.get("message") is not None:
         message_text = payload["message"] + "\n\n"
 
+    alert_status = ""
+    if payload.get("state") is not None:
+        if payload.get("state") == "alerting":
+            alert_status = ":alert: " + "**" + payload["state"].upper() + "**" + "\n\n"
+        elif payload.get("state") == "ok":
+            alert_status = ":squared_ok: " + "**" + payload["state"].upper() + "**" + "\n\n"
+        else:
+            alert_status = "**" + payload["state"].upper() + "**" + "\n\n"
```

Comment:
We should still have a generic badge here. So that we can use the constant here too.